exercise_02/getTotalWordsFromString.js

```diff
@@ -3,3 +3,17 @@
  * input: string (sentence)
  * output: number
  */
+
+/**
+ *
+ * @param {string} sentence
+ * @returns number
+ */
+
+const getTotalWordsFromString = function (sentence) {
+  const numberOfWords = sentence.length;
```

Comment:
This method is not returning the number of words contained in a sentence

FIX: needs work